### ISSUE TITLE
CosmosAsync is missing in Spotify 1.1.79.763

### ISSUE THREAD

**Issue by rlaphoenix** [khanhas/spicetify-cli#1493]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: Spotify for Windows 1.1.79.763.gc2965cdf
- Spicetify version: 2.9.0
```

### 📝 Description

1. Install specified versions
2. Open dev tools
3. Run `Spicetify.CosmosAsync`

Expecting `Spicetify.CosmosAsync` to exist, but returns `undefined`.

**Comment by CharlieS1103** [comment]
Run spicetify upgrade

**Comment by rlaphoenix** [comment]
Ahh, I see it was fixed with v2.9.1, thanks.
